presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixTableProperties.java

```diff
@@ -109,6 +110,11 @@ public PhoenixTableProperties()
                         TTL,
                         "Number of seconds for cell TTL.  HBase will automatically delete rows once the expiration time is reached.",
                         null,
+                        false),
+                stringProperty(
+                        DATA_BLOCK_ENCODING,
+                        "The block encoding algorithm to use for Cells in HBase blocks. Options are: PREFIX, DIFF, FAST_DIFF, ROW_INDEX_V1, and others.",
```

Comment:
Did we miss `None` and `PREFIX_TREE` encoding technique ?

Reply:
Can we add the same in the doc too

Reply:
PREFIX_TREE is not finished in HBase, nobody should use it. Yep. None is missing.

Reply:
Updated.